### DISCUSSION TITLE
Best language models

### DISCUSSION THREAD

**Discussion by rakuri255** [rakuri255/UltraSinger#63] [Q&A]
Post your additional language model here, that works good for your language.

[Your feedback for this model]
[Language] [huggingface link]

**Comment by stripu** [comment]
For romanian, all the models on huggingface are using a variation of facebook's wav2vec2, so you'll get very similar results, but personally i'm using infinitejoy's version, as it's the most recent one. Results are around 95% accuracy on speech recognition and 90% accuracy on hyphenation.

Romanian - https://huggingface.co/infinitejoy/wav2vec2-large-xls-r-300m-romanian

**Comment by RobUmf** [comment]
Had a Request for a few Arabic songs.
Found this, It work. I don't know Arabic. He said it was OK.
https://huggingface.co/jonatasgrosman/wav2vec2-large-xlsr-53-arabic
